src/extension-runners/chromium.js

```diff
@@ -217,7 +231,8 @@ export class ChromiumExtensionRunner {
         if (msg.type === 'webExtReloadAllExtensions') {
           const devExtensions = await getAllDevExtensions();
           for (var ext of devExtensions) {
-            reloadExtension(ext.id);
+            await reloadExtension(ext.id);
+            ws.send(JSON.stringify({ type: 'webExtReloadExtensionComplete' }));
```

Comment:
This should be done after the loop.

And although it does currently not matter, I hope that you are aware of the fact that previously the `reloadExtension` was run in parallel, but after the change they are run sequentially. In general, if you want to keep running them in parallel, use use an `Array`'s `map` method to generate an array of promises, and then use `Promise.all`.